src/labels/CustomLabels.labels

```diff
@@ -4921,6 +4921,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Percent</shortDescription>
         <value>Percent</value>
     </labels>
+    <labels>
+        <fullName>commonPermissionErrorMessage</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Page-level error for database level security check</shortDescription>
```

Comment:
```suggestion
        <shortDescription>Page-level error for database-level security check</shortDescription>
```

Reply:
@melissabarber just pushed up that change.